go.sh

```diff
@@ -71,4 +83,4 @@ case "${LOG_FORMAT_NAME}" in
     ;;
 esac
 
-eval "/usr/local/openresty/nginx/sbin/nginx -g \"daemon off;\""
\ No newline at end of file
+eval "/usr/local/openresty/nginx/sbin/nginx -g \"daemon off;\""
```

Comment:
It should totally be an exec to correctly handle sigs but it was probably on a Monday and the daemon off flag didn't?